app/novelty/compute_novelty.py

```diff
@@ -3,10 +3,27 @@
 import httpx
 import numpy as np
 import traceback
-
-from .known import find_known_results
-from .extr_smile_molpro_by_id import mol_to_smile_molpro
-from .mol_similarity import find_nearest_neighbors
+import json
+from known import find_known_results
+from extr_smile_molpro_by_id import mol_to_smile_molpro
+from mol_similarity import find_nearest_neighbors
```

Comment:
These three lines are not package imports, but local. They fail to be imported.

Reply:
Got it. I've imported the two necessary functions into the compute_novelty.py file to make it simpler. There won't be a need to import the three files anymore. I'll keep them in the repo in case there is future requirement of use.

Reply:
Understood. I have removed the files from the repo.